controllers/netbot.js

```diff
@@ -3832,6 +3832,32 @@ class netBot extends ControllerBot {
         })
         break;
       }
+      case "getDNSProfile": {
```

Comment:
follow this to move get/set logic to the same sensor file

https://github.com/firewalla/firewalla/blob/9e21f28efa341e0ecaebff81d4302dacc03675e4/sensor/GuardianSensor.js#L50